src/Handler/CurlFactory.php

```diff
@@ -2,6 +2,7 @@
 namespace M6Web\Bundle\GuzzleHttpBundle\Handler;
 
 use GuzzleHttp\Handler\CurlFactory as GuzzleCurlFactory;
+// @TODO use something else since EasyHandle class is @internal (=> BC is not ensured)
```

Comment:
can we use [TransferStats](https://github.com/guzzle/guzzle/blob/master/src/TransferStats.php) instead ?

Reply:
sorry, I don't know... I don't even know what's the purpose of using `EasyHandle` 😥 

I just put this `@TODO` to warn other developers about using classes tagged with `@internal`... because I think some developers don't know it

Reply:
Well they both give access to the result of `curl_getinfo`, but the TransfertStats thing was added to guzzle after (only 3 months :D) the EasyHandle hack thing.

Reply:
I created an issue to tackle this in a separated PR 
=> https://github.com/M6Web/GuzzleHttpBundle/issues/67